test/linter_test.dart

```diff
@@ -123,10 +123,16 @@ void defineLinterEngineTests() {
 
     group('groups', () {
       test('factory', () {
-        expect(new Group('Style').custom, isFalse);
-        expect(new Group('Pub').custom, isFalse);
+        expect(new Group('style').custom, isFalse);
+        expect(new Group('pub').custom, isFalse);
+        expect(new Group('errors').custom, isFalse);
         expect(new Group('Kustom').custom, isTrue);
```

Comment:
Intentional. Just not a built-in.  ("Kustom" is a play on *custom*.)

The company **Kustom** also made great (and sometimes hilarious) amplifiers [back in the day](http://www.kustomcollector.com/id1.html).